### PR TITLE
Fix some -Wfloat-conversion warnings (Pt. 6)

### PR DESCRIPTION
**Pull request by Holzhaus**
Almost done...

### CODE REVIEW

File: src/effects/lv2/lv2effectprocessor.cpp [hunk 1]
```diff
@@ -117,7 +117,7 @@ void LV2EffectProcessor::process(const ChannelHandle& inputHandle,
     } 
 
     for (int i = 0; i < m_parameters.size(); i++) {
-        m_params[i] = m_parameters[i]->value();
+        m_params[i] = static_cast<CSAMPLE>(m_parameters[i]->value());
```
Comment (daschuer) [inline src/effects/lv2/lv2effectprocessor.cpp:120]: We don't know the use here so casting to float would fit better.

File: src/effects/lv2/lv2effectprocessor.cpp [hunk 2]
```diff
@@ -142,7 +142,7 @@ LV2EffectGroupState* LV2EffectProcessor::createGroupState(const mixxx::EnginePar
     LilvInstance* handle = pState->lilvIinstance();
     if (handle) {
         for (int i = 0; i < m_parameters.size(); i++) {
-            m_params[i] = m_parameters[i]->value();
+            m_params[i] = static_cast<CSAMPLE>(m_parameters[i]->value());
```
Comment (daschuer) [inline src/effects/lv2/lv2effectprocessor.cpp:145]: Same here, cast too float.

**Review by daschuer — CHANGES_REQUESTED** [review]
Some comments.

**Comment by daschuer** [comment]
This has conflicts now.

**Comment by Holzhaus** [comment]
The conflicts have been resolved.

**Review by daschuer — APPROVED** [review]
LGTM, Thank you.

**Comment by daschuer** [comment]
Waiting for CI

**Comment by Holzhaus** [comment]
Merge?

**Pull request merged**